SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -299,9 +308,13 @@ - (void)didEnterStateConnected {
 
     // If we have security managers, add them to the proxy
     if (self.configuration.streamingMediaConfig.securityManagers != nil) {
-        SDLLogD(@"Adding security managers");
+        SDLLogD(@"Adding security managers from streamingMedia configuration");
         [self.proxy addSecurityManagers:self.configuration.streamingMediaConfig.securityManagers forAppId:self.configuration.lifecycleConfig.appId];
```

Comment:
Even though this is necessary, I think that deprecating taking security managers in the `SDLStreamingMediaConfiguration` is still necessary.

Reply:
`securityManagers` property and `init` methods cannot be deprecated easily in `SDLStreamingMediaConfiguration` due to the dependencies of `SDLStreamingAudioLifecycleManager` and `SDLStreamingVideoLifecycleManager` and related classes.  This should a new proposal written to refactor to code, like the refactor of `SDLProxy`